x-pack/plugins/infra/public/alerting/metric_threshold/components/alert_flyout.tsx

```diff
@@ -22,43 +18,26 @@ interface Props {
   setVisible: React.Dispatch<React.SetStateAction<boolean>>;
 }
 
-interface KibanaDeps {
-  notifications: NotificationsStart;
-  http: HttpStart;
-  docLinks: DocLinksStart;
-  application: ApplicationStart;
-}
-
 export const AlertFlyout = (props: Props) => {
   const { triggersActionsUI } = useContext(TriggerActionsContext);
-  const { services } = useKibana<KibanaDeps>();
 
-  return (
-    <>
-      {triggersActionsUI && (
-        <AlertsContextProvider
-          value={{
-            metadata: {
-              currentOptions: props.options,
-              series: props.series,
-            },
-            toastNotifications: services.notifications.toasts,
-            http: services.http,
-            docLinks: services.docLinks,
-            capabilities: services.application.capabilities,
-            actionTypeRegistry: triggersActionsUI.actionTypeRegistry,
-            alertTypeRegistry: triggersActionsUI.alertTypeRegistry,
-          }}
-        >
-          <AlertAdd
-            addFlyoutVisible={props.visible!}
-            setAddFlyoutVisibility={props.setVisible}
-            alertTypeId={METRIC_THRESHOLD_ALERT_TYPE_ID}
-            canChangeTrigger={false}
-            consumer={'infrastructure'}
-          />
-        </AlertsContextProvider>
-      )}
-    </>
+  const AddAlertFlyout = useMemo(
+    () =>
+      triggersActionsUI &&
+      triggersActionsUI.getAddAlertFlyout({
+        consumer: 'logs',
```

Comment:
should the consumer be `infrastructure`?

Reply:
Great catch about consumer!